src/Shape.java

```diff
@@ -0,0 +1,9 @@
+
+    public interface Shape {
+        double getArea();
+
+        public double calculateAreaOfTheFigure(double a , double h);
```

Comment:
what is the purpose of this method `calculateAreaOfTheFigure`?
There is already a method  `getArea()`